starter_code/routes/movies.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const router = express.Router();
+const Movie = require("../model/Movie");
+
+router.get("/movies", (req, res) => {
+  Movie.find().then((movies) => {
+    res.render("movies", { movies: movies });
```

Comment:
quick tip here:
in modern javasight you could have made this a little shorter :)

```js
res.render("movies", {movies})
```